docarray/utils/find.py

```diff
@@ -23,6 +23,16 @@ class _FindResult(NamedTuple):
     scores: AnyTensor
 
 
+class FindResultBatched(NamedTuple):
+    documents: List[DocList]
+    scores: AnyTensor
```

Comment:
This should be `List[AnyTensor]` or `List[np.ndarray]` like in the qdrant and weaviate PRs. The reason is that for batched search operations, especially if filtering is involved, not every query is guaranteed to yield the same number of results. If this happens, then the scores cannot be stacked up into one 2-axis array/tensor. Thus the different type.

Reply:
Just made these changes let me know if something else is needed.